src/FrameHelper.h

```diff
@@ -60,9 +63,10 @@ namespace frame_helper
             //this is a convenience function to undistort a frame
             //call setCalibrationParameter first
             //see cv::remap 
-            void undistort(const base::samples::frame::Frame &src,base::samples::frame::Frame &dst);
-
+            void undistort(const base::samples::frame::Frame &src, base::samples::frame::Frame &dst, 
+                    UndistortAlgorithm undistort_algorithm = UNDISTORT_CUBIC);
 
+            static cv::InterpolationFlags undistortAlgorithmToCV(frame_helper::UndistortAlgorithm const& undistort_algorithm);
```

Comment:
This does NOT need to be a static method. Just define it as a static function in the `.cpp` without mentioning it in .hpp

It should AFAIK allow you to remove the `#include <imgproc.hpp>`